TASVideos.Data/Entity/Game/Game.cs

```diff
@@ -60,6 +60,13 @@ public static IQueryable<Game> ForSystemCodes(this IQueryable<Game> query, IEnum
 			: query;
 	}
 
+	public static IQueryable<Game> ForLetter(this IQueryable<Game> query, string? code)
```

Comment:
If you are only going to support a single character, the param should be a char type.  However, maybe simpler is just to use .StartsWith(), and then this extension method isn't needed